GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -485,7 +469,7 @@ public virtual bool IsPathProjected(string virtualPath, out string fileName, out
                 childName: fileName,

```

Comment:
Maybe add a comment above the call to `GetProjectedFolderEntryData` that mentions `null` will be returned if a path's parent folder is not included.